pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -164,6 +166,14 @@ func (k *KubeData) NewNodeInformer(informerFactory informers.SharedInformerFacto
 					ips = append(ips, ip.String())
 				}
 			}
+			// Add IP that is used in OVN for some traffic on mp0 interface
+			ip, err := findOvnMp0IP(node.Annotations)
```

Comment:
TBH I don't know, this is the only case I've found so far, I don't know if it's better to target wide or precise .. you think I should add every IP in the range ?

Reply:
maybe get comments from @amorenoz

Reply:
I don't know if there ovn-k8s puts other networks in the annotations or, if it does, there is a straightforward way of getting the node's IP on such network. Maybe @astoycos.

Also note that this could be confusing as this IP is also used for Services backed by NodePorts IIUC